phobos/scripts/run_pipeline.py

```diff
@@ -58,7 +60,11 @@ def main(args):
         print("Success rate: {:.2f} %".format(pipeline.get_coverage(
             phases=phases if not args.verify else ["process", "test", "deploy"],
             allow_na=args.allow_na_in_verify)*100), file=sys.stderr)
+        log.info("Success rate: {:.2f} %".format(pipeline.get_coverage(
+            phases=phases if not args.verify else ["process", "test", "deploy"],
+            allow_na=args.allow_na_in_verify)*100))
```

Comment:
Wouldn't this lead to double printing if BASE_LOG_LEVEL is info?

Reply:
indeed

Reply:
Wird rausgeschmissen